pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/store/TableViewLoadDataStoreImpl.java

```diff
@@ -92,7 +92,11 @@ public synchronized CompletableFuture<Void> removeAsync(String key) {
     public synchronized Optional<T> get(String key) {
         String msg = validateTableView();
         if (StringUtils.isNotBlank(msg)) {
-            throw new IllegalStateException(msg);
+            if (msg.equals(SHUTDOWN_ERR_MSG)) {
+                return Optional.empty();
```

Comment:
can you explain why we need to return empty here?

Reply:
Without this change, `testCloseAfterLoadingBundles` would always fail.

```
2024-09-26T22:29:35,236 - INFO  - [main:ServiceUnitStateChannelImpl] - Started ownership cleanup for the inactive broker:localhost:58804
2024-09-26T22:29:35,244 - ERROR - [main:ServiceUnitStateChannelImpl] - Failed to override inactiveBroker:localhost:58804 ownership serviceUnit:public/default/0x50000000_0x60000000 orphanData:ServiceUnitStateData[state=Owned, dstBroker=localhost:58804, sourceBroker=localhost:58783, splitServiceUnitToDestBroker=null, force=true, timestamp=1727360975026, versionId=3]. totalCleanupErrorCnt:1
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: This load store tableview has been shutdown
...
```